public String divide(Integer a, Integer b){

    try {
        Integer resultado = a/b;
        String resultadoConvert= String.valueOf(resultado);
        return resultadoConvert; 

    } catch(Exception  e) {
        System.debug('The following exception has occurred: ' + e.getMessage());
    }  

}